bindings/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,388 @@
+# Contributing to PyMongoArrow
+
+Contributions to PyMongoArrow are always encouraged. Contributions can be as simple
+as minor tweaks to the documentation. Please read these guidelines
+before sending a pull request.
+
+## Bugfixes and New Features
+
+Before starting to write code, look for existing
```

Comment:
-[tickets](https://jira.mongodb.org/browse/ARROW) or [create
-one](https://jira.mongodb.org/browse/ARROW) for your specific issue or
+[tickets in our JIRA project](https://jira.mongodb.org/browse/ARROW) or create
+one for your specific issue or

Reply:
Fixed